src/tools/launcher/python_launcher.cc

```diff
@@ -30,6 +30,10 @@ static constexpr const char* WINDOWS_STYLE_ESCAPE_JVM_FLAGS = "escape_args";
 
 ExitCode PythonBinaryLauncher::Launch() {
   wstring python_binary = this->GetLaunchInfoByKey(PYTHON_BIN_PATH);
+
+  // Rlocation returns the original path if python_binary is an absolute path.
+  python_binary = this->Rlocation(python_binary, true);
```

Comment:
But here, `python_binary` will just be `python` if nothing is specified.. I need to fix this.

Reply:
Thanks, let me know when the PR is ready for another review.

Reply:
Thanks for catching the problem! Please take a look again. ;)

Reply:
Oh, just realized I had a typo. It should be `GetBinaryPathWithoutExtension(bash_binary) != L"bash")`, `GetBinaryPathWithoutExtension` just strips out the ending `.exe` if there is any. I wanted to check if bash_binary is already `bash` or `bash.exe`. Similar for the python case.

Reply:
Will add comment for this!